ingestion/tests/cli_e2e/test_cli_oracle.py

```diff
@@ -128,12 +130,13 @@ def expected_filtered_table_includes() -> int:
 
     @staticmethod
     def expected_filtered_table_excludes() -> int:
-        return 29
+        return 30
 
     @staticmethod
     def expected_filtered_mix() -> int:
         return 43
 
+    @pytest.mark.xfail
```

Comment:
add the the link to the issue in the reason

Reply:
example: https://github.com/open-metadata/OpenMetadata/blob/a455bbe0ce08fe7f015754b64ac70d17e125cd32/ingestion/tests/integration/postgres/test_usage.py#L51